src/bundle/Resources/views/themes/admin/url_wildcard/list.html.twig

```diff
@@ -9,6 +10,21 @@
     }
 %}
 
+<section class="container">
+	{{ form_start(form_list, {'attr': {'class': 'ibexa-search-links-form'}}) }}
+	<div class="ibexa-search-links-form__input-wrapper">
+		{{ form_widget(form_list.searchQuery, { attr: {
+			'placeholder': 'url_wildcard.search.placeholder'|trans|desc('Search for URL wildcards'),
+		}}) }}
```

Comment:
We can probably delete `_url_wildcard_list_searchQuery_widget` and add `has_search` parameter here:
```suggestion
		{{ form_widget(form_list.searchQuery, { 
		    attr: {
			placeholder: 'url_wildcard.search.placeholder'|trans|desc('Search for URL wildcards'),
		    },
		    has_search: true,
		}) }}
```
Can you confirm? 
(AFAICS, `_url_wildcard_list_searchQuery_widget` was not present in the original PR, if it was introduced in the original PR, the above suggestions do not apply. 🙂 )